app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str):
+    file = command.split(" ")
+    if file[1] != file[2]:
```

Comment:
you should check the input command

Reply:
I don't understand, but I check it, split the command, and fit.

Reply:
<img width="919" alt="image" src="https://user-images.githubusercontent.com/72562232/189642954-6c8d0150-7ff4-47ee-817c-6b862892d82e.png">
check if the input command is `cp`